data/transactions/logic/debugger.go

```diff
@@ -202,12 +202,12 @@ func valueDeltaToValueDelta(vd *basics.ValueDelta) basics.ValueDelta {
 
 // parseCallStack initializes an array of CallFrame objects from the raw
 // callstack.
-func (d *DebugState) parseCallstack(callstack []int) []CallFrame {
+func (d *DebugState) parseCallstack(callstack []frame) []CallFrame {
 	callFrames := make([]CallFrame, 0)
 	lines := strings.Split(d.Disassembly, "\n")
-	for _, pc := range callstack {
+	for _, fr := range callstack {
 		// The callsub is pc - 3 from the callstack pc
-		callsubLineNum := d.PCToLine(pc - 3)
+		callsubLineNum := d.PCToLine(fr.retpc - 3)
```

Comment:
Maybe I am confused here, so `fr.retpc` points to pc after callsub, right? Then `-3` has 2 parts contributed:
- 2 for `uint16` label
- 1 for `callsub` byte representation
Sounds correct?